### ISSUE TITLE
Test fails when using pytest 5.0.*

### ISSUE THREAD

**Issue by gwtnb** [cupy/cupy#2466]
```
% pytest -x tests/cupy_tests/core_tests/test_raw.py
==================================================================================================================== test session starts =====================================================================================================================
platform linux -- Python 3.6.9, pytest-5.0.1, py-1.8.0, pluggy-0.12.0
rootdir: /path/to/cupy, inifile: setup.cfg
plugins: xdist-1.28.0, forked-1.0.2
collected 7 items                                                                                                                                                                                                                                            

tests/cupy_tests/core_tests/test_raw.py ..F

========================================================================================================================== FAILURES ==========================================================================================================================
_____________________________________________________________________________________________________________ TestRaw.test_get_function_failure ______________________________________________________________________________________________________________

self = <cupy_tests.core_tests.test_raw.TestRaw testMethod=test_get_function_failure>

    def test_get_function_failure(self):
        # in principle this test is better done in test_driver.py, but
        # this error is more likely to appear when using RawModule, so
        # let us do it here
        with pytest.raises(cupy.cuda.driver.CUDADriverError) as ex:
            self.mod2.get_function("no_such_kernel")
>       assert 'CUDA_ERROR_NOT_FOUND' in str(ex)
E       AssertionError: assert 'CUDA_ERROR_NOT_FOUND' in '<ExceptionInfo CUDADriverError tblen=7>'
E        +  where '<ExceptionInfo CUDADriverError tblen=7>' = str(<ExceptionInfo CUDADriverError tblen=7>)

tests/cupy_tests/core_tests/test_raw.py:160: AssertionError
============================================================================================================= 1 failed, 2 passed in 1.02 seconds =============================================================================================================
```

**Comment by emcastillo** [comment]
Reproduced!

**Comment by emcastillo** [comment]
Fixed by #2473
